### PR TITLE
Site update

### PR DESCRIPTION
**Pull request by sorenrood**
### What changed?
- added a disclaimer that we are no longer working on project
- removed all references to our newsletter
- removed all references to our API
- removed marketing lingo that was unnecessary "start your 7 day free trial now"

### Why?
I wanted the site to be accurate. I don't want anyone to think we are still maintaining the project. Personal closure.

### CODE REVIEW

**Review by notjustinshaw — DISMISSED** [review]
This looks good!

**Comment by notjustinshaw** [comment]
Screen Shot 2022-03-28 at 2 36 25 PM
is this the text we want

File: src/components/banner/banner.tsx [hunk 1]
```diff
@@ -17,17 +17,11 @@ export default function Banner() {
               <p className="ml-3 font-light text-white truncate">
                 <span className="md:hidden">We shipped our newsletter!</span>
```
Comment (notjustinshaw) [inline src/components/banner/banner.tsx]: @sorenrood -- can we change this copy as well?

Comment (sorenrood) [reply src/components/banner/banner.tsx]: Just fixed it, thanks!

**Comment by sorenrood** [comment]
penguin_dance

**Review by notjustinshaw — APPROVED** [review]
LGTM
LGTM 👍🏻

**Pull request merged**